src/check_mysql_buffer_pool_usage.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+"""InnoGames Monitoring Plugins - MySQL InnoDB Buffer Pool Check
+
+This script monitors the usage of the InnoDB buffer pool by issuing a query that calculates
+the percentage of the buffer pool currently utilized. It raises alerts based on defined thresholds.
+
+Warning at 90% usage.
+
+Copyright (c) 2020 InnoGames GmbH
+"""
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+from argparse import ArgumentParser
+from mysql.connector import (
+    connect,
+    DatabaseError,
+)
+
+
+def parse_args():

```

Comment:
```suggestion
def parse_args() -> argparse.Namespace:
```
Mind that the import must be added or adjusted.